6.Exception-handling/ex.js

```diff
@@ -0,0 +1,23 @@
+function division(){
+  try {
+    console.log (a)

```

Comment:
this must be done separately, in the first task you need to catch an error that the variable is not declared. In the second task it is necessary to generate an error.
Due to the fact that you have two tasks done together, you do not generate an error.